hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/hedera/embedded/AbstractEmbeddedHedera.java

```diff
@@ -192,8 +196,16 @@ public void start() {
                 .addressBook(toPbjAddressBook(addressBook))
                 .build());
         ((CommittableWritableStates) writableStates).commit();
+
+        final var writableRosterStates = state.getWritableStates(RosterService.NAME);
+        final WritableRosterStore writableRosterStore = new WritableRosterStore(writableRosterStates);
+        if (!hedera.isRosterLifecycleEnabled()) {
+            writableRosterStore.putActiveRoster(buildRoster(addressBook), 0);
+            ((CommittableWritableStates) writableRosterStates).commit();
```

Comment:
I thought this code is on develop.

Reply:
It was until recently, I brought this back in order to make the `hedera.setRosterHistory()` call below work.